ci/generate_docs.sc

```diff
@@ -253,6 +265,23 @@ def buildDocs(buildDir: Path, docsDir: Path, checkedRepoDir: Path) = {
   generateVersionedDocs(buildDir, versionedDocsDirs)
 }
 
+def publishToGithub(buildDir: Path, remote: String): Unit = {
+  println("Publishing docs to github")
+  %('git, 'clone, "-b", "gh-pages", "--single-branch", remote, "marathon-gh-pages")(buildDir)
+  val generatedDocsDir = buildDir/'docs/'_site
```

Comment:
This is a little confusing to me. When and where are the docs generated? Should `publishToGithub` just receive the path to the docs?